verilog/tools/preprocessor/verilog_preprocessor_test.sh

```diff
@@ -14,6 +14,18 @@
 # limitations under the License.
 
 # Find input files
+MY_RELATIVE_INCLUDED_FILE_1="myinclude_1.txt"
```

Comment:
I find it usually more readable to combine the assignment and readonly in one go; it then reads like `const` in C/C++.
```bash
readonly MY_RELATIVE_INCLUDED_FILE_1="myinclude_1.txt"
```

(also good to apply to the other variables in this file where possible).